Tests/Engine/InvokeScriptAnalyzer.tests.ps1

```diff
@@ -406,6 +406,13 @@ Describe "Test CustomizedRulePath" {
                     Pop-Location
                 }
             }
+            
+            It "resolves rule preset when passed in via pipeline" {
+                $warnings = 'CodeFormattingStroustrup' | ForEach-Object {
```

Comment:
this seems overly complicated, isn't this just:
```
$warnings = Invoke-ScriptAnalyzer -ScriptDefinition 'if ($true){}' -Settings CodeFormattingStroustrup
$warnings.Count | Should -Be 1
$warnings.RuleName | Should -Be 'PSUseConsistentWhitespace'
```
all the pipes and where's aren't needed and only add complexity